pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -36,7 +36,7 @@ U.S. Web Design System source code is written in Sass, a powerful stylesheet lan
 
 We recommend using `uswds-compile`, a USWDS tool [hosted on GitHub](https://github.com/uswds/uswds-compile), as a quick way to get up and running with Sass compilation for USWDS.
 
-This tool provides simple Gulp 4 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
+This tool provides simple Gulp 5 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
```

Comment:
**question**: Curious if you all think the specificity of the major version number is needed here. Can we just replace it with "Gulp"?

```suggestion
This tool provides simple Gulp functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
```

Reply:
I like this idea, less of a chance it gets missed in the future.

Reply:
Replaced "Gulp 5" with "Gulp" in [4426600](https://github.com/uswds/uswds-site/pull/2750/commits/44266000175d05ec29501f17b4187efab1b15f53)